packages/features/bookings/lib/handleSeats/reschedule/owner/moveSeatedBookingToNewTimeSlot.ts

```
@@ -3,22 +3,38 @@ import { cloneDeep } from "lodash";
 const moveSeatedBookingToNewTimeSlot = async (
   rescheduleSeatedBookingObject: RescheduleSeatedBookingObject,
   seatedBooking: SeatedBooking,
   eventManager: EventManager,
-  loggerWithEventDetails: ReturnType<typeof createLoggerWithEventDetails>
+  traceContext?: TraceContext
 ) => {
+  const spanContext = traceContext
+    ? DistributedTracing.createSpan(traceContext, "move_seated_booking_to_new_time_slot", {
+        meta: {
+          bookingId: seatedBooking.id,
+          rescheduleUid,
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)